modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/feature/flag/FeatureFlagsBag.java

```diff
@@ -83,13 +84,18 @@ public boolean isEnabled(String key) {
 			return featureFlag.isEnabled();
 		}
 
-		if (_log.isInfoEnabled()) {
-			_log.info(
-				StringBundler.concat(
-					"Feature flag ", key, " is not available for company ",
-					_companyId));
+		String scope = "instance";
+
+		if (_companyId != CompanyConstants.SYSTEM) {
+			scope = "system";
 		}
 
+		_log.error(
+			StringBundler.concat(
+				"Feature flag ", key, " is not available for company ",
+				_companyId, ". Is it a ", scope, "-scoped feature flag?"),
+			new RuntimeException());
```

Comment:
Two things here:
1. Is this exception needed? You should be able to just log an error message by itself.
2. Not sure about this wording. I think it would be simpler to just log the feature flag key and companyId being queried, since the nuances of this policy will be documented in Confluence and all we need to do is cause a CI failure with enough info to figure out what needs to change.